tests/test_idpi/test_curl.py

```diff
@@ -19,14 +19,13 @@ def test_curl(data_dir):
     set_origin_xy(ds, ref_param="HHL")
 
     geo = ds["HHL"].attrs["geography"]
-    dlon = geo["iDirectionIncrementInDegrees"]
     dlat = geo["jDirectionIncrementInDegrees"]
     nj = geo["Nj"]
     lat_min = geo["latitudeOfFirstGridPointInDegrees"]
 
     deg2rad = np.pi / 180
     rlat = get_grid_coords(nj, lat_min, dlat, "y") * deg2rad
```

Comment:
As done in the simplifications in `pot_vortic.py` in this PR:
```suggestion
    deg2rad = np.pi / 180
    rlat = get_grid(ds["HHL"].attrs["geography"]).rlat * deg2rad
```